package.json

```diff
@@ -112,7 +110,23 @@
     "eslint-plugin-react": "^7.32.2",
     "msw": "^1.0.0",
     "prettier": "^2.7.1",
+    "snyk": "^1.1293.1",
     "ts-jest": "^27.0.0",
     "typescript": "^5.0.4"
+  },
+  "resolutions": {
```

Comment:
Are all these resolution specifications needed? For example this matches the axios version is explicitly stated above, does specifying it here as well have any effect?

Reply:
You're right that some of these resolutions may seem redundant if the version is already specified elsewhere, such as for axios. However, we included them in the resolutions section to ensure that the specified versions are enforced across all dependencies, especially if there are sub-dependencies that could resolve to different or conflicting versions. This approach helps prevent potential compatibility issues that might arise if different parts of the project use conflicting versions of the same package.

Reply:
As i understand resolutions, then that is an override to dependencies and how they are resolved. Any entries added here will make updating the versions in the dependencies sections. I think we need to be very careful before adding anything to resolutions section, since it can make our lives much harder later on.